config/initializers/audit_events.rb

```diff
@@ -7,5 +7,5 @@
 # Audit::Subscribe.listen :internal, :quest_user
 
 Rails.application.reloader.to_prepare do
-  Audit::Subscribe.listen(:moderator, :internal) unless Rails.env.test?
+  Audit::Subscribe.listen(:moderator, :internal, :admin_api) unless Rails.env.test?
```

Comment:
It took me longer than I'm proud to admit to find this line in our codebase 😅 

But I'm happy I learned more about how the [ActiveSupport Instrumentation API](https://guides.rubyonrails.org/active_support_instrumentation.html#creating-custom-events) works!

Reply:
Whoa, TIL!